test/raygun_send_test.js

```diff
@@ -27,7 +27,7 @@ test("send basic", {}, function (t) {
   var client = new Raygun.Client().init({
     apiKey: API_KEY,
   });
-  client.send(new Error(), {}, function (response) {
+  client.sendWithCallback(new Error(), {}, function (response) {
```

Comment:
The old `raygun_send_test` still uses the legacy `sendWithCallback` method, this test fine can be removed when the `sendWithCallback` method is finally removed.